README.md

```diff
@@ -84,3 +84,24 @@ You can add it as a regular dependency to the build of your app:
   <version>$version</version>
 </dependency>
 ```
+
+## Code Style
+
+To ensure that new code formatting matches the requirements for Pull Requests,
+the Maven Checkstyle plugin can be used to create a report listing possibly coding 
+style violations.
+
+The [Checkstyle](https://checkstyle.org) configuration is currently in a very early stage and so at present
+it checks for empty blocks, extra white space, padding and empty lines. 
+
```

Comment:
I would reword the above paragraphs to the following:

> Contributors can check for code-style violations in their code by running the [Checkstyle](https://checkstyle.org) Maven goal. The checkstyle configuration is currently in a very early stage and only checks for empty blocks, extra white space, padding and empty lines.

Reply:
Yes, this sounds much friendlier, so this scentence is updated accordingly.